docker/poetry.lock

```diff
@@ -146,7 +146,7 @@ description = "Backport of PEP 654 (exception groups)"
 optional = false
 python-versions = ">=3.7"
 groups = ["dev"]
-markers = "python_version < \"3.11\""
+markers = "python_version == \"3.10\""
```

Comment:
@hakonhagland I have no idea what this does. It might not allow using Python 3.11 anymore. Should be worried about this change?